push/model/src/main/java/org/jboss/aerogear/unifiedpush/message/Criteria.java

```diff
@@ -82,6 +83,37 @@ public void setVariants(List<String> variants) {
         this.variants = variants;
     }
 
+    /**
+     * Helper method to check if only categories are applied. Usful in GCM land, where we use topics
+     */
+    @JsonIgnore
+    public boolean isCategoryOnly() {
```

Comment:
@secondsun I thought about this, and will move this into a (GCM Topic) Util class.